### DISCUSSION TITLE
Bug in Namma Yatri App

### DISCUSSION THREAD

**Discussion by ShivangiSingh123** [nammayatri/nammayatri#3755] [General]
There is a bug in Namma Yatri App, when the user clicks on pickup location, the text box should be empty but it already contains a text "current location" that should be removed as soon as the user starts entering the pickup location.
![IMG_20231017_013537](https://github.com/nammayatri/nammayatri/assets/30458459/ea22aa8e-e9db-4ce1-a309-cc0aacb7b80d)

**Comment by ashishsingh101** [comment]
@ShivangiSingh123 Thanks for bringing this to our attention. We've revamped it to ensure a smoother and quicker experience.
